src/System.IO.Pipelines/tests/PipeWriterCopyToAsyncTests.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Pipelines.Tests
+{
+    public class PipeWriterCopyToAsyncTests
```

Comment:
Add tests for argument validation on stream.CopyToAsync() and pipeReader.CopyToAsync(). Ex pass in a canceled cts, null args, etc.